docs/data-guide/big-data/non-relational-data-content.md

```diff
@@ -34,7 +35,7 @@ The following diagram shows an example with two column families, `Identity` and
 
 Unlike a key/value store or a document database, most column-family databases physically store data in key order, rather than by computing a hash. The row key is considered the primary index and enables key-based access via a specific key or a range of keys. Some implementations allow you to create secondary indexes over specific columns in a column family. Secondary indexes let you retrieve data by columns value, rather than row key.
 
-On disk, all of the columns within a column family are stored together in the same file, with a specific number of rows in each file. With large data sets, this approach creates a performance benefit by reducing the amount of data that needs to be read from disk when only a few columns are queried together at a time.
+On disk, all of the columns within a column family are stored together in the same file, with a specific number of rows in each file. With large data sets, this approach creates a performance benefit by reducing the amount of data that needs to be read from disk when only a few columns are queried together at a time. On the other hand, columnar data stores may not be a good option when you need to query multiple fields from each row. Also, writing into columnar stores takes longer because for inserting a new entity, you need to fill each column one by one (in relational databases, it can be done in one operation)
```

Comment:
Reverting late addition.  There isn't enough justification in this text to include it.  You can easily include multiple columns as part of responses in columnar data stores (they are called column families and are introduced above. The performance bit, may have been better in the paragraph below which was already started to address write options.

I'll just leave it out for now.

```suggestion
On disk, all of the columns within a column family are stored together in the same file, with a specific number of rows in each file. With large data sets, this approach creates a performance benefit by reducing the amount of data that needs to be read from disk when only a few columns are queried together at a time.
```